src/Resources/skeleton/stimulus/Controller.tpl.php

```diff
@@ -0,0 +1,18 @@
+import { Controller } from '@hotwired/stimulus';
+
+/*
+* The following line makes this controller "lazy": it won't be downloaded until needed
+* See https://github.com/symfony/stimulus-bridge#lazy-controllers
+*/
+/* stimulusFetch: 'lazy' */
+export default class extends Controller {
+<?= $targets ? "    static targets = $targets\n" : "" ?>
+<?php if ($values) { ?>
+    static values = {
+<?php foreach ($values as $value): ?>
+        <?= $value['name'] ?>: <?= $value['type'] ?>,
```

Comment:
just I think the last `,` is to much no? I mean after the latest value type is written

Reply:
That's a trailing comma, it's a good practice in Javascript